prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -582,7 +597,8 @@ void rxr_pkt_handle_data_copied(struct rxr_ep *ep,
 	rx_entry->bytes_copied += data_size;
 
 	if (rx_entry->total_len == rx_entry->bytes_copied) {
-		if (rxr_get_base_hdr(pkt_entry->pkt)->flags & RXR_REQ_WAIT_RECEIPT) {
+		if (rxr_get_base_hdr(pkt_entry->pkt)->flags & RXR_REQ_WAIT_RECEIPT ||
```

Comment:
can we set ```RXR_DELIVERY_COMPLET_REQUESTED``` to rx_entry->fi_flags for other type of rx_entry? In that case, we just need to check ```rx_entry->fi_flags & RXR_DELIVERY_COMPLETE_REQUESTED``` here.

Reply:
Set `RXR_DELIVERY_COMPLETE_REQUESTED` to `rx_entry->rxr_flags`, mark as resolved.